### ISSUE TITLE
Grijze cq disabled titel in Expandable Header

### ISSUE THREAD

**Issue by kad-leeffs** [dso-toolkit/dso-toolkit#2175]
In de viewer willen we titels en onderliggende teksten uitgrijzen, of disablen. We willen de nadruk leggen op relevant teksten, juist niet op de teksten die grijs zijn. Op dit moment is het niet mogelijk om de kleur van een titel aan te passen, anders dan 'black' of default.

Dit moet verwerkt worden in de **Expandable Header**

Mogelijke oplossingen:
- Attribuut `color` uitbreiden met grijs: grey
- Ander custom attribuut vullen, zoals `disabled`, die zorgt voor de grijze uitstraling.

**Comment by kad-leeffs** [comment]
@tfrijsewijk Hierbij. Heb je hier voldoende aan?

**Comment by tfrijsewijk** [comment]
Ik heb er dit van geimproviseerd:


![Image](https://github.com/dso-toolkit/dso-toolkit/assets/9696261/99c6374b-bc61-4c09-9f1a-74ddbcd54aab)

**Comment by kad-leeffs** [comment]
Thanks @tfrijsewijk . Hij doet het, alleen dan wel met letterlijk het attribuut op de expandable heading zetten, niet als input variabele. Dus:
`<dso-expandable-heading suppressed ...` ipv `<dso-expandable-heading [suppressed='true'] ...`

**Comment by tfrijsewijk** [comment]
Klopt, de styling zit op het attribuut. Ik zal de volgende keer de [property naar het attribuut laten synchroniseren](https://stenciljs.com/docs/properties#reflect-properties-values-to-attributes-reflect) :+1:

Het was een beetje laat 😄
